sos/report/plugins/lxd.py

```diff
@@ -44,6 +44,17 @@ def setup(self):
         snap_list = self.exec_cmd('snap list lxd')
         if snap_list["status"] == 0:
             self.add_cmd_output("lxd.buginfo", pred=lxd_pred)
+
+            if not self.get_option("all_logs"):
+                self.add_copy_spec([
+                    '/var/snap/lxd/common/lxd/logs/*.log',
+                    '/var/snap/lxd/common/lxd/logs/*/*.log',
+                    '/var/snap/lxd/common/lxd/logs/*/*.conf',
```

Comment:
Hm... when re-reading the particular paths to collect: are not config files important enough to be collected every time? They are often older than logfiles so when logs are over size limit, sos report would collect newest logs but not configs (when --all-lags is *not* used).

Does not it deserve to mention configs as an extra `copy_path` in the `else` branch? (or what do config files do in a `log` directory..? :-o)

Reply:
The .conf files would be collected with the below line in `else` as seen in https://paste.ubuntu.com/p/n3PYq46XBS/

happy to separate it out, if that is preferred

Reply:
If you know the whole `/var/snap/lxd/common/lxd/logs/**` in `else` branch will always be below default size limit, no need to separate. If you are unsure and need to collect configs while optionally truncate older logs, let separate the configs.

Reply:
I think I'll separate it, I think that makes more sense